scripts/nesi_scripts/run_adv_im_obs_maui.sl

```diff
@@ -62,12 +62,13 @@ do
         time python $IMPATH/calculate_ims.py $path_eventBB a -o $path_event_out -np 40 -i $event -r $event -t  o -e -a $adv_IM_model --OpenSees_path $opensees_bin
         # test for completion 
         res=`python $gmsim/workflow/scripts/verify_adv_IM.py $path_event_out $adv_IM_model`
-        exit 
         if [[ $? == 0 ]];then
```

Comment:
Better too to compare res rather than $0

Reply:
Given that verify_adv_IM returns a string, and exits with a return code, this should be fine

Reply:
I prefer return code, unless we changed the output of the verify_adv_IM to be numerical, it creates extra dependencies.

Reply:
Ah yeah, I'm less keen to use $0 as it's led to bugs in the past in case lines are added between the call and the result.

Reply:
Unless, we get verify to print out the error and return a code?

Similar to how validate_vm is used?

Reply:
*$?

Reply:
adding a variable to store the return code solves that issue.

Reply:
Put the return code into its own variable, and use that in the check. Put them on adjacent lines and if anyone tries to put a line between them their PR gets denied

Reply:
Also a comment that the return code is needed for the test. So that someone else coming to this code knows why they shouldn't just jam a new line in there

Reply:
You could even put them on the same line with a ";"

Reply:
Too slow Jason, Jonney got there first

Reply:
was just talking to james saying that xD

Reply:
I said it 8 minutes ago...